feat: New Grouping docs (#2173)
getsentry/sentry-docs: src/platforms/common/data-management/event-grouping/sdk-fingerprinting.mdx

---
title: "SDK Fingerprinting"
sidebar_order: 0
redirect_from:
  - ../../../enriching-error-data/additional-data/fingerprint-override/
---

In supported SDKs, you can override the Sentry default grouping passing the `fingerprint` attribute an array of strings. This works similar to the [server side functionality](../server-side-fingerprinting/) which is always available and
can achieve similar results.

<PlatformContent includePath="set-fingerprint/basic">

## Basic Example

> In the most basic of cases values are passed directly:

Reviewer: ```suggestion
In the most basic of cases, values are passed directly:
```

</PlatformContent>

> Variable substitution can be used to fill in dynamic values into the fingerprint that are normally computed on the server. For instance the value `{{ default }}` can be added to add the entire normally generated grouping hash into the fingerprint. These values are the same as for server side fingerprinting. See [Variables](../server-side-fingerprinting/#variables) for more information.

Reviewer: ```suggestion
You can use variable substitution to fill dynamic values into the fingerprint that are normally computed on the server. For instance, the value `{{ default }}` can be added to add the entire normally generated grouping hash into the fingerprint. These values are the same as for server side fingerprinting. See [Variables](../server-side-fingerprinting/#variables) for more information.
```
